CRM/Core/BAO/SchemaHandler.php

```diff
@@ -920,7 +920,7 @@ public static function getInUseCollation(): string {
    */
   public static function databaseSupportsUTF8MB4(): bool {
     if (!isset(\Civi::$statics[__CLASS__][__FUNCTION__])) {
-      \Civi::$statics[__CLASS__][__FUNCTION__] = stripos(self::getInUseCollation(), 'utf8mb4') === TRUE;
+      \Civi::$statics[__CLASS__][__FUNCTION__] = stripos(self::getInUseCollation(), 'utf8mb4') === 0;
```

Comment:
@eileenmcnaughton what if instead it was `!== FALSE`

Reply:
Then the function would be wrong. The exact value returned when the string starts with 'utf8mb4' is 0 - because that is the position it holds in the string